coa/pkg/apis/v1alpha2/providers/keylock/memory/memory.go

```diff
@@ -39,43 +54,54 @@ func toMemoryKeyLockProviderConfig(config providers.IProviderConfig) (MemoryKeyL
 	return ret, err
 }
 
-func (mslp *MemoryKeyLockProvider) Init(config providers.IProviderConfig) error {
+func (gml *GlobalMemoryKeyLock) Init(config providers.IProviderConfig) error {
+	sLog.Info("Init GlobalMemoryKeyLock")
```

Comment:
If user load GlobalMemoryKeyLock in multiple managers with different configurations, what's the expected behavior?

Reply:
<img width="695" alt="image" src="https://github.com/user-attachments/assets/b83a871f-002a-43b8-9fbb-319cc482faa8">
Seems we should introduce "GlobalMemoryKeyLock" and its config in this place. Not in very MemoryKeyLockProvider.

Reply:
added shared MemoryKeyLock